src/tools/pub/package-layout.md

```diff
@@ -59,8 +59,15 @@ enchilada/
     style.css
 {% endprettify %}
 
-\* The `.dart_tool/` directory and `.packages` file exist after you've run
-   `pub get`. Don't check them into source control.
+{{site.alert.secondary}}
+  **Version note:** Prior to Dart 2.17, `pub get` also
+  generated a `.packages` file. That generated file has been
+  replaced by `package_config.json`.
+{{site.alert.end}}
+
+\* The `.dart_tool/` directory and `package_config.json`
+   files exist after you've run `pub get`.
+   Don't check them into source control.
```

Comment:
This one we could mention `package_config.json` but I'm not sure it is necessary since it is part of the `.dart_tool/` directory.

```suggestion
\* The `.dart_tool/` directory exists after you've run `pub get`.
   Don't check it into source control.
```